src/RubberDuckyParking/parking/templates/parking/host_details.html

```diff
@@ -1,12 +1,22 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Host Details</title>
-</head>
-<body>
-<h1>Host Profile for user <em>{{host_id}}</em></h1>
-<p>Here we can make a host profile that displays their parking lots, manage their parking spots, manage their attendants,
-see their transactions, view money report, etc</p>
-</body>
-</html>
\ No newline at end of file
+{% extends 'base.html' %}
+{% block head %} {% endblock %}
+
+{% block content%}
+
+{% if user.is_authenticated %} <p>and is a host</p>
```

Comment:
this is not working rn with pr 'f_user_model_rework_2'

Reply:
should be working just fine, I'll pull and make sure.